commands/cli.go

```diff
@@ -58,6 +58,29 @@ func parseDurationParam(paramValue string) (*time.Time, error) {
 }
 
 // Nuke it all!!!
+func cloudNuke(c *cli.Context) error {
+	if c.NArg() > 0 {
+		providers := []string{"aws", "azure", "gcp"}
+		provider := c.Args().Get(0)
+		if collections.ListContainsElement(providers, provider) {
+			switch provider {
+			case "aws":
+				return awsNuke(c)
+			case "azure":
+				return UnsupportedProviderError{
+					Name: "azure",
+				}
+			case "gcp":
+				return UnsupportedProviderError{
+					Name: "aws",
+				}
+			}
+		}
+	}
+
+	return UnsupportedProviderError{}
```

Comment:
If there are no args, you should prob just show the help text (`urfave/cli` has some method to do that).